econml/utilities.py

```diff
@@ -697,3 +790,518 @@ def predict(self, Xt):
         t = Xt[:, -self.n_T:]
         predictions = [self.model_list[np.nonzero(t[i])[0][0]].predict(X[[i]]) for i in range(len(X))]
         return np.concatenate(predictions)
+
+def _safe_norm_ppf(q, loc=0, scale=1):
+    prelim = loc.copy()
+    if hasattr(prelim, "__len__"):
+        if np.any(scale > 0):
+            prelim[scale > 0] = scipy.stats.norm.ppf(q, loc=loc[scale > 0], scale=scale[scale > 0])
+    elif scale > 0:
+        prelim = scipy.stats.norm.ppf(q, loc=loc, scale=scale)
+    return prelim
+
+class StatsModelsWrapperOLD:
```

Comment:
Do we really want to call this OLD? My code was not really written for direct integration. Also I thought you wanted to change the name var_weight to sample_vars, which I agree with.

Reply:
This legacy class is now moved to the test file (where it's used to compare the new implementation to the direct statsmodels implementation), and `var_weight` is renamed to `sample_var`.